gradle.properties

```diff
@@ -8,15 +8,19 @@ eventuateMavenRepoUrl=https://dl.bintray.com/eventuateio-oss/eventuate-maven-rel
 springBootVersion=1.4.7.RELEASE
 springBootCdcVersion=2.1.1.RELEASE
 
-eventuateClientVersion=0.21.1.RELEASE
-eventuateLocalVersion=0.30.2.RELEASE
+eventuateLocalVersion=0.15.0-SNAPSHOT
 eventuateUtilVersion=0.1.0.RELEASE
 
 springCloudContractDependenciesVersion=2.0.0.RELEASE
 springDependencyManagementPluginVersion=1.0.3.RELEASE
 
 springCloudSleuthVersion=2.0.2.RELEASE
 dockerComposePluginVersion=0.6.6
-
-version=0.21.0-SNAPSHOT
+eventuateCdcVersion=0.1.0-SNAPSHOT
```

Comment:
Eventuate CDC should only be used in tests (src/tests) or as an image version

Reply:
I will fix